### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Sharmishatha**
added comments for code review

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,3 +2,5 @@
 Test Repository
 
 Sharmishtha Change the file.
+
+Code review must be done.
```
Comment (Sharmishatha) [inline README.md:6]: This has to be changed.

Comment (Sharmishatha) [inline README.md:6]: This has to be changed.

**Review by Sharmishatha — COMMENTED** [review]
Added comments.

File: README.md [hunk 2]
```diff
@@ -2,3 +2,5 @@
 Test Repository
 
 Sharmishtha Change the file.
```
Comment (Sharmishatha) [inline README.md:4]: Correct spellings.

File: README.md (hunk 1 above)
Comment (Sharmishatha) [inline README.md:6]: hi

File: README.md [hunk 3]
```diff
@@ -2,3 +2,5 @@
 Test Repository
```
Comment (subhash-singh) [inline README.md:2]: hjj

**Review by subhash-singh — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (subhash-singh) [inline README.md:4]: update

**Review by subhash-singh — CHANGES_REQUESTED** [review]

**Pull request merged**